FullDemo/playkitdemo/src/main/java/com/kaltura/playkitdemo/LoadControlSetup.java

```diff
@@ -0,0 +1,121 @@
+    //private static int startBitrate;
+
+    private static boolean loaded = false;
+
+    // Call once as soon as the app loads, in Application.onCreate()
+    public static void load() {
+        if (loaded) {
+            return;
+        }
+
+        AsyncTask.execute(() -> {
+            synchronized (LoadControlSetup.class) {
+                if (!loaded) {
+                    OkHttpClient client = new OkHttpClient();
+                    Request request = new Request.Builder().url(GET_UICONF_DATA).build();
+                    try {
+                        Response response = client.newCall(request).execute();
+                        populateLoadControlsMembers(response);
+                    } catch (IOException e) {
+                        e.printStackTrace();
+                    }
+                    loaded = true;
+                }
+            }
+        });
+    }
+
+    public static Integer getExperimentId() {
+        return experimentId;
+    }
+
+    private static void populateLoadControlsMembers(Response response) throws IOException {
+        if (response != null && response.body() != null) {
+            String uiconfFile = response.body().string();
+            JsonParser parser = new JsonParser();
+            JsonObject uiconf = parser.parse(uiconfFile).getAsJsonObject();
+            if (uiconf != null) {
+                JsonPrimitive config = uiconf.getAsJsonPrimitive("config");
+                if (config != null) {
+                    JsonObject configJsonObj = parser.parse(config.getAsString()).getAsJsonObject();
+                    if (configJsonObj != null) {
+                        Gson gson = new Gson();
+                        LoadControlModel loadControlModel = gson.fromJson(configJsonObj, LoadControlModel.class);
+                        if (loadControlModel != null) {
+                            initialBuffer = loadControlModel.getInitialBuffer();
+                            afterRebuffer = loadControlModel.getAfterRebuffer();
+                            bufferLength = loadControlModel.getBufferLength();
+                            experimentId = loadControlModel.getExperimentId();
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+    // Call after creating a player with loadPlayer()
+    public static void apply(Player player) {
+        player.getSettings().setPlayerBuffers(
+                new LoadControlBuffers().setMinPlayerBufferMs(LoadControlSetup.initialBuffer).
+                        setMaxPlayerBufferMs(LoadControlSetup.bufferLength).
+                        setMinBufferAfterReBufferMs(LoadControlSetup.afterRebuffer));
+    }
+
+    class LoadControlModel {
+
+        private Integer experimentId;
+        private Integer initialBuffer;
+        private Integer afterRebuffer;
+        private Integer bufferLength;
+        private Integer startBitrate;
+
+        public LoadControlModel() {}
```

Comment:
Is it possible to use `public final` fields instead of private + getters, or Gson won't allow it?